examples/gpt-4o-vs-4o-mini/README.md

```diff
@@ -1,13 +1,35 @@
-This example shows how you can use promptfoo to generate a side-by-side eval of multiple prompts to compare gpt-4o and gpt-4o-mini outputs.
+# gpt-4o-vs-4o-mini (Comparing GPT-4o and GPT-4o-Mini)
 
-Configure in `promptfooconfig.yaml`. Run with:
+## Quick Start
 
+1. Initialize this example by running:
+
+```bash
+npx promptfoo@latest init --example gpt-4o-vs-4o-mini
 ```
-promptfoo eval
+
+2. Navigate to the newly created `gpt-4o-vs-4o-mini` directory:
+
+```bash
+cd gpt-4o-vs-4o-mini
+```
+
+3. Set an OpenAI API key directly in your environment:
+
+```bash
+export OPENAI_API_KEY="your_openai_api_key"
 ```
 
-Full command-line equivalent:
+Alternatively, you can set the API key in a `.env` file:
 
 ```
-promptfoo eval --prompts prompts.txt --providers openai:gpt-4o openai:gpt-4o-mini
+OPENAI_API_KEY=your_openai_api_key
 ```
+
+4. Run the evaluation with:
+
+```bash
+npx promptfoo@latest eval -o output.html --no-cache
```

Comment:
good except they should just run `promptfoo view`. the output.html file -> browser is old and barely supported